src/main/java/duke/DeadlineInput.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import duke.Deadline;
+
+public class DeadlineInput extends InputHandler {
+
+    public DeadlineInput(Ui ui, TaskList taskList) throws DukeException {
+        super(ui, taskList);
+    }
+
+    @Override
```

Comment:
I think you could refactor your toString method here! or possibly store the instantiated objects as variables instead!